kpi/views/environment.py

```diff
@@ -141,4 +129,8 @@ def get(self, request, *args, **kwargs):
             settings.STRIPE_PUBLIC_KEY if settings.STRIPE_ENABLED else None
         )
         data['social_apps'] = social_apps
+        data['mfa_available_to_user'] = MfaAvailableToUser.objects.filter(
```

Comment:
This name `mfa_available_to_user` gave me hard time. Maybe because of my memory of a golden-fish, but I always have to look at the code why it returns `false` if the the per-user availability list is empty. 
At the first look, it's not obvious that it tells if user is part of the list or not. 

Do you think we could find something more obvious. (I don't have a good idea).

Reply:
Good point! I didn't think to update that after changing what was getting returned from `/environment`. I went with `mfa_per_user_availability`, like the admin setting, let me know if that's clearer.